packages/frontend-shared/src/components/Select.cy.tsx

```diff
@@ -170,19 +168,19 @@ describe('<Select />', () => {
     it('renders all of the slots', () => {
       const vSlots = {
         'item-body': () => 'Item Body',
-        'item-prefix': () => <Icon icon={IconHeart} data-testid="item-prefix" />,
-        'item-suffix': () => <Icon icon={IconHeart} data-testid="item-suffix" />,
+        'item-prefix': () => <div data-testid="item-prefix"></div>,
+        'item-suffix': () => <div data-testid="item-suffix"></div>,
```

Comment:
Having values in those slots can help make sense of the video recording

```suggestion
        'item-prefix': () => <div data-testid="item-prefix">ip</div>,
        'item-suffix': () => <div data-testid="item-suffix">is</div>,
```